compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -741,7 +743,6 @@ setAttentionVectorDistributionConfig(IREE::GPU::TargetAttr target,
 
   if (!schedule) {
     LDBG("Failed to deduce Attention schedule");
-    return failure();
```

Comment:
should probably revive back return failure

Reply:
Sounds good